pkg/monitor/monitor.go

```diff
@@ -264,6 +264,11 @@ func (monitor *agentMonitor) getAgentCRD(crdName string) *v1beta1.AntreaAgentInf
 }
 
 func (monitor *agentMonitor) createAgentCRD(crdName string) (*v1beta1.AntreaAgentInfo, error) {
+	ovsVersion := monitor.GetOVSVersion()
+	ovsConnected := true
+	if ovsVersion == "" {
+		ovsConnected = false
+	}
```

Comment:
how about `ovsConnected := ovsVersion != ""`?

Reply:
Thanks for the suggestion. Have updated it accordingly.